app/services/taxonomy/show_page.rb

```diff
@@ -67,5 +67,20 @@ def associated_taxons
     def taxon_deletable?
       taxon.content_id != GovukTaxonomy::ROOT_CONTENT_ID
     end
+
+    def email_subscribers
+      @email_subscribers ||= begin
+        email_lists = []
+
+        begin
+          email_lists = Services.email_alert_api.find_subscriber_list(links: { taxon_tree: [taxon.content_id] })
+        rescue GdsApi::BaseError, SocketError => e
```

Comment:
Is it worth adding a test for this condition?

Reply:
What would be the best way to do that?  A test which just unconditionally asserts that a `SocketError` should be raised is weird; so ideally the test would make email-alert-api inaccessible (if it currently is)

Reply:
To test for `SocketError`, you can do something like this:

```
stub_request(:get, "https://email-alert-api.test.gov.uk/subscriber-lists")
   . to_raise(SocketError)
```

Reply:
Probably the simplest way is to let email-alert-api return a 500 error:

```
stub_request(:get, "https://email-alert-api.test.gov.uk/subscriber-lists")
   .to_return(body: [].to_json, status: 500)
```

This should cause gds-api-adapters to raise a `GdsApi::ServerError` (or something).

@thomasleese sure that works? Looks like it's mixing RSpec and gds-api-adapters methods.

Reply:
Seems to work